plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts

```diff
@@ -7,23 +7,83 @@ import { runInstrumentedFunction } from '../../utils'
 import { KafkaJSIngestionConsumer } from '../kafka-queue'
 import { eachBatch } from './each-batch'
 
+// TODO: remove once we've migrated
 export async function eachMessageAsyncHandlers(message: KafkaMessage, queue: KafkaJSIngestionConsumer): Promise<void> {
     const clickHouseEvent = JSON.parse(message.value!.toString()) as RawClickHouseEvent
     const event = convertToIngestionEvent(clickHouseEvent)
 
+    await Promise.all([
+        runInstrumentedFunction({
+            server: queue.pluginsServer,
+            event: event,
+            func: () => queue.workerMethods.runAppsOnEventPipeline(event),
+            statsKey: `kafka_queue.process_async_handlers_on_event`,
+            timeoutMessage: 'After 30 seconds still running runAppsOnEventPipeline',
+            teamId: event.teamId,
+        }),
+        runInstrumentedFunction({
+            server: queue.pluginsServer,
+            event: event,
+            func: () => queue.workerMethods.runWebhooksHandlersEventPipeline(event),
+            statsKey: `kafka_queue.process_async_handlers_webhooks`,
+            timeoutMessage: 'After 30 seconds still running runWebhooksHandlersEventPipeline',
+            teamId: event.teamId,
+        }),
+    ])
+}
+
+// TODO: remove once we've migrated
+export async function eachBatchAsyncHandlers(
+    payload: EachBatchPayload,
+    queue: KafkaJSIngestionConsumer
+): Promise<void> {
+    await eachBatch(payload, queue, eachMessageAsyncHandlers, groupIntoBatches, 'async_handlers')
```

Comment:
tangential: the signature of these eachBatch functions is very convoluted imo. Having eachBatchAsyncHandlers just perform the batching, and running the processing directly would be nicer imo

no action required.